web/api/badges/web_buffer_svg.c

```diff
@@ -688,15 +688,15 @@ static inline void calc_colorz(const char *color, char *final, size_t len, calcu
 // colors
 #define COLOR_STRING_SIZE 100
 
-void buffer_svg(BUFFER *wb, const char *label, calculated_number value, const char *units, const char *label_color, const char *value_color, int precision, int scale, uint32_t options) {
+void buffer_svg(BUFFER *wb, const char *label, calculated_number value, const char *units, const char *label_color, const char *value_color, int precision, int scale, uint32_t options, int fixed_width_lbl, int fixed_width_val) {
     char    value_color_buffer[COLOR_STRING_SIZE + 1]
             , value_string[VALUE_STRING_SIZE + 1]
             , label_escaped[LABEL_STRING_SIZE + 1]
             , value_escaped[VALUE_STRING_SIZE + 1]
             , label_color_escaped[COLOR_STRING_SIZE + 1]
             , value_color_escaped[COLOR_STRING_SIZE + 1];
 
-    double label_width, value_width, total_width, height = 20.0, font_size = 11.0, text_offset = 5.8, round_corner = 3.0;
+    double label_width = fixed_width_lbl, value_width = fixed_width_val, total_width, height = 20.0, font_size = 11.0, text_offset = 5.8, round_corner = 3.0;
```

Comment:
Hi @underhood ,

I know that we are talking here about numbers(`double` and `int`), but it is possible some compilers will generate warnings that in this code we are giving directly `int` variables to `double` variables, please, can you add some cast here?